### PR TITLE
Home work lesson 18

### PR DESCRIPTION
**Pull request by sdmnc**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </head>
 
 <body>
-    <script src = "script2.js">
+    <script src = "script3.js">
```
Comment (IvanCeban) [inline index.html]: Better pust the script in the head section

File: script3.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+["Megan", "Kate", "Beth", "Eve"],
+["Barbie", "Ken", "Marusia", "Lea"]
+
+]
+const sortedArray = new Array()
+const myArray = new Array()
+for (let i = 0; i < company.length; i++) {
+    if (i > 0) {
+        //  console.log(company[i])
+        let name = company[i]
```
Comment (IvanCeban) [inline script3.js]: clean up the code. You're not using "name" variable

File: script3.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        sortedArray.push(company[i])
```
Comment (IvanCeban) [inline script3.js]: this array is not sorted, so naming it "sortedArray" sounds confusing.

File: script3.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+const company = [["Dev", "QA", "HR", "Finance"],
+["John", "Bob", "Samanta", "Peter"],
+["Silivia", "Jane", "Monica", "Henry"],
+["Megan", "Kate", "Beth", "Eve"],
+["Barbie", "Ken", "Marusia", "Lea"]
+
+]
+const sortedArray = new Array()
+const myArray = new Array()
+for (let i = 0; i < company.length; i++) {
```
Comment (IvanCeban) [inline script3.js:45]: This loop seems redundant. I bet you can get rid of it and use only the loop from line 54

**Review by IvanCeban — CHANGES_REQUESTED** [review]

File: script3.js (hunk 4 above)
Comment (sdmnc) [reply script3.js:45]: Sorry I don't get it. I can manually remove const company = [["Dev", "QA", "HR", "Finance"], from object and after that I don't need that "for" is that what you mean?

**Pull request merged**